clang/test/SemaSYCL/sycl-device.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
+// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify -DHOST %s
 // RUN: %clang_cc1 -verify -DNO_SYCL %s
```

Comment:
Please add a comment here about what this test does (now that we are modifying it).